hadoop-hdds/docs/content/shell/BucketCommands.zh.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 桶命令
+summary: 用桶命令管理桶的生命周期
+weight: 3
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone shell 提供以下桶命令：
+
+  * [创建](#创建)
+  * [删除](#删除)
+  * [查看](#查看)
+  * [列举](#列举)
+
+### 创建
+
+用户使用 `bucket create` 命令来创建桶。
+
+***参数：***
+
+| 参数名                      |  说明                                |
+|--------------------------------|-----------------------------------------|
+| -g, \-\-enforcegdpr            | 可选，如果设置为 true 则创建符合 GDPR 规范的桶，设置为 false 或不指定则创建普通的桶|
```

Comment:
There is a --bucketKey option missing from the original English document. I've opened HDDS-2937 to fix it.